test/__snapshots__/serialize.test.ts.snap

```diff
@@ -12503,6 +12503,7 @@ __fnls_AWS.Lambda = __fnls_AWS_Lambda;
 __f19.kind = \\"$AWS.EventBridge.putEvent\\";
 var __fnls_AWS_EventBridge = {putEvents: __f19};
 __fnls_AWS.EventBridge = __fnls_AWS_EventBridge;
+__fnls_AWS.SDK = undefined;
```

Comment:
I think me refactoring somehow broke the serialize tests. This used to be something like:

```
__fnls_AWS.SDK = {};
```

Reply:
Based on the synth output in the new test below, I don't imagine the current state will actually work at runtime in lambda.  In theory undefined or void are fine until its actually used. 

1. Have you tried this in lambda, I see the tests are not actually running on the PR for whatever reason.
2. Is it the proxy or the importing that is breaking the serialization?
3. Is it different if you do something like 
```ts
const cloudWatchClient = $AWS.SDK.CloudWatch;
new Function(..., async () => {
    cloudWatchClient.describeAlarms(...)
})
```

Reply:
I tried adding a test where I use a referenced method outside the 

```ts
const {describeAlarms} = $AWS.SDK.CloudWatch;

serialize( () => {describeAlarms(...)})
```

And it said that either SDK or CloudWatch is undefined.